ui/narrative/methods/annotate_contigs/display.yaml

```diff
@@ -19,7 +19,7 @@ suggestions:
             []
     methods:
         related:
-            []
+            [annotate_contigset, annotate_domains_in_a_genome, annotate_plant_transcripts, reannotate_microbial_genome]
```

Comment:
@nlharris: Per Eric, we don't show these IDs in UI or anywhere else. Do you think it's worth putting them here? If yes then for what reason?